src/utils/logging.hpp

```diff
@@ -47,24 +47,15 @@ void AssertFailed(const char *file_name, int line_num, const char *expr, const A
   std::terminate();
 }
 
-// TODO (antonio2368): Replace with attribute [[likely]] when it's supported by
-// compilers
 #define MG_ASSERT(expr, ...) \
   LIKELY(!!(expr))           \
```

Comment:
can you replace this with [[likely]] and remove the TODO?

Reply:
on it!